.github/workflows/lint-and-format.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         with:
           node-version: 16
           cache: npm
+      - uses: google/wireit@setup-github-actions-caching/v1
 
       - run: npm ci
+      - run: npm run lint
```

Comment:
Consider adding a lint-and-format build script

or just make `npm run lint` run both in parallel, as format checking feels like a linting